File: javascript/strings.js
const words = "Hello World! My name is Rick Parker.";

words.charAt(6); // "W"
words.endsWith("."); // true
words.includes("Parker"); // true
words.indexOf("W"); // 6
words.repeat(2); // "Hello World! My name is Rick Parker.Hello World! My name is Rick Parker."
words.replace("World", "People"); // "Hello People! My name is Rick Parker."
words.slice(13); // "My name is Rick Parker."
words.split(" "); // ["Hello", "World!", "My", "name", "is", "Rick", "Parker."]
words.startsWith("H"); // true
words.toLowerCase(); // "hello world! my name is rick parker".
words.toUpperCase(); // "HELLO WORLD! MY NAME IS RICK PARKER."
words.trim(); // "Hello World! My name is Rick Parker."

File: python/strings.py
message = "hello World! My name is John Doe."

message.capitalize()  # Hello world! my name is john doe.
message.casefold()  # hello world! my name is john doe.
message.center(50)  # _________hello World! My name is John Doe._________
message.count("e")  # 3
message.encode()  # b"hello World! My name is John Doe."
message.endswith(".")  # True
message.find("!")  # 11
message.index("!")  # 11
message.isalpha()  # False
message.isdecimal()  # False
message.isdigit()  # False
message.islower()  # False
message.isnumeric()  # False
message.isupper()  # False
message.lower()  # hello world! my name is john doe.
message.replace("Doe", "Parker")  # hello World! My name is John Parker.
message.rfind("e")  # 31
message.rindex("e")  # 31
message.split()  # ["hello", "World!", "My", "name", "is", "John", "Doe."]
message.startswith("h")  # True
message.strip()  # hello World! My name is John Doe.
message.upper()  # HELLO WORLD! MY NAME IS JOHN DOE.
